app/src/main/java/com/github/se/project/ui/profile/ProfileInfo.kt

```diff
@@ -50,12 +51,12 @@ fun ProfileInfoScreen(
             horizontalArrangement = Arrangement.SpaceBetween) {
               Row(Modifier.padding(horizontal = 4.dp)) {
                 IconButton(
-                    onClick = { /* Navigate to calendar screen */},
+                    onClick = { navigationActions.navigateTo(Screen.EDIT_SCHEDULE) },
```

Comment:
Should have an if condition to navigate to the student equivalent, but again can be added later